src/cpp/threadpool.cpp

```diff
@@ -31,9 +26,7 @@ ThreadPool::ThreadPool(unsigned int num_threads)
     // Create worker threads
 	if (num_threads < 1) num_threads = 1; // Force a minimum of 1 thread
     for (unsigned int i = 0; i < num_threads; i++) {
-        #ifdef TIME_THREADPOOL
-        std::cout << "Kicking off thread " + std::to_string(i) << std::endl;
-        #endif
+        log_data(LLDeveloper, "Kicking off thread " + std::to_string(i));
```

Comment:
If the previous request is accepted, can we change this message to `"Kicking off thread i of n"`

Reply:
see previous